tests-functional/tests/test_local_pairing.py

```diff
@@ -296,7 +296,7 @@ def test_pairing_three_devices(self, backend_new_profile, backend_factory):
         self.make_contacts(user_accepted, bob1)
         self.send_contact_request_and_wait_for_signal_to_be_received(user_pending, bob1)
         message_id = self.send_contact_request_and_wait_for_signal_to_be_received(user_declined, bob1)
-        bob1.wakuext_service.decline_contact_request(message_id)
+        bob1.wakuext_service.decline_contact_request(message_id, user_declined.public_key)
```

Comment:
Shouldn't you also test the opposite case, ie when _not_ passing the pubKey, the test should fail?